parent/pom.xml

```diff
@@ -622,6 +622,37 @@
                         </execution>
                     </executions>
                 </plugin>
+
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-checkstyle-plugin</artifactId>
+                    <version>3.0.0</version>
```

Comment:
I would recommend moving plugin's version to parent/pom.xml with `maven.checkstyle.plugin.version` name and use variable here instead.

Reply:
Pert, thank you for your comments. 
Agree here. Fixed